### DISCUSSION TITLE
通过电子邮件升级为专业设备，没有接收到邮件验证码

### DISCUSSION THREAD

**Discussion by yiiiiiiiiiiiiiiiiii** [getlantern/lantern-forum-cn#1106] [General]
电子邮件地址：yiiiiii2024@163.com

**Comment by MaxMKMax** [comment]
@yiiiiiiiiiiiiiiiiii 您好, 请您提供我支付单据. 谢谢您.

**Comment by yiiiiiiiiiiiiiiiiii** [comment]
支付账单找不到了，我另外一台设备还能正常用，有截图，烦请处理


| |
13187886985
|
|
***@***.***
|


---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年12月31日 20:13 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***> ,
***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 通过电子邮件升级为专业设备，没有接收到邮件验证码 (Discussion #1106) |

@yiiiiiiiiiiiiiiiiii 您好, 请您提供我支付单据. 谢谢您.

**Comment by yiiiiiiiiiiiiiiiiii** [comment]
| |
13187886985
|
|
***@***.***
|


---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年12月31日 20:13 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***> ,
***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 通过电子邮件升级为专业设备，没有接收到邮件验证码 (Discussion #1106) |

@yiiiiiiiiiiiiiiiiii 您好, 请您提供我支付单据. 谢谢您.
